Klarna.Rest/Klarna.Rest.Core/Model/Enum/ShippingMethod.cs

```diff
@@ -5,12 +5,14 @@ namespace Klarna.Rest.Core.Model.Enum
     /// </summary>
     public enum ShippingMethod
     {
-        PickupStore, 
```

Comment:
This change will brake the backward compatibility

Reply:
Added back enum